src/components/AmountForm.tsx

```diff
@@ -47,8 +50,13 @@ const NUM_PAD_CONTAINER_VIEW_ID = 'numPadContainerView';
 const NUM_PAD_VIEW_ID = 'numPadView';
 
 function AmountForm(
+<<<<<<< HEAD
     {value: amount, currency = CONST.CURRENCY.USD, extraDecimals = 0, errorText, onInputChange, onCurrencyButtonPress}: AmountFormProps,
     forwardedRef: ForwardedRef<TextInput>,
+=======
+    {value: amount, currency = CONST.CURRENCY.USD, extraDecimals = 0, errorText, onInputChange, onCurrencyButtonPress, isCurrencyPressable = true}: AmountFormProps,
+    forwardedRef: ForwardedRef<BaseTextInputRef>,
+>>>>>>> af1026e (Merge pull request #36497 from shubham1206agra/fix-currency-ui)
```

Comment:
I _think_ to solve it you'd just need to copy over the `isCurrencyPressable = true` param to line 54, but def understand if you don't wanna worry about it and just make the issues NABs

Reply:
That is one thing, but also BaseTextInputRef doesn't exist here yet and then I found a couple other files with missing imports so I didn't want to risk it since I'm not sure what is not on staging yet. I'd have to correct all of those files :yikes:

Reply:
ah okay yeah fair enough! :nevermind: